README.md

```diff
@@ -131,34 +131,23 @@ All fields are optional. `title, `desc`, and `id` are strings.
 
 ##### `textIndex` key (optional, NEW/experimental)
 
-A path to a text index file in the ZIP which contains a seperate full text search index,
+A path to a text index file in the ZIP which contains a full text search index and metadata information about the pages,
 ideally in the `text/` directory.
 
-#### 4) `pages.csv` (optional)
-
-The list of pages, specified in a CSV format.
-Each row should match the 'Page' object and contain at least the following columns: `url`, `date`, `title`, `id`
-The `url` and `date` are required. The date should be in ISO 8601 format.
-
 #### 5) `text/` directory (experimental)
 
 Contains a text index as a newline-deliminted JSON format, eg.
 Currently the text index is only used if it is enabled by setting the `textIndex` entry in `datapackage.json`
 
 ```
-{"url": "https://example.com/", "title": "Example Domain": "text": "Example Domain\nThis domain is for..."}
-{"url": "https://example.com/another", "title": "Another Example", "text": "Some other text for this page..."}
+{"id": 0, "url": "https://example.com/", "title": "Example Domain": "text": "Example Domain\nThis domain is for..."}
+{"id": 1, "url": "https://example.com/another", "title": "Another Example", "text": "Some other text for this page..."}
 ```
```

Comment:
Change examples to random id to match the new uuid?

Reply:
UUID? This is nice! We sometimes use UUID V5 when we want to have a repeatable/always the same UUID based on a certain seed https://docs.python.org/3/library/uuid.html#uuid.uuid5 . By passing as first argument the constant `uuid.NAMESPACE_URL` and then your current "url" value as the second argument you get that. Does not change the actual string/sequence/pattern but allows you to always get a consistent, same UUID string when using the URL as seed, so maybe a future idea. Or also, just a comment you can totally dismiss because it may add processing time! and on a huge list that adds up! :) Happy you went this route. Good night!

Reply:
Hm, that's an interesting idea, and tempting, but probably not a good idea, as this would imply that a URL or URL + TS are fully unique, which we can't guarantee. For example, we could have two captures of the same URL at same time that are different (different cookies, different locations, etc..), and this would imply that they're fixed.
Having a generic id string is also more flexible, as can import from systems that already have a page ID.

Maybe we should make it clear that the id is just a unique string, we're using uuid but it doesn't have to be that?

Reply:
Totally! Makes sense, I thought URL were unique but of course they do not need to be like that. Thanks!!